pkg/operator/cluster/cluster.go

```diff
@@ -133,7 +130,8 @@ func (c *Cluster) createClientAccess(clusterInfo *cephmon.ClusterInfo) error {
 	defer conn.Shutdown()
 
 	// create a user for rbd clients
-	username := "client.rook-rbd-user"
+	name := fmt.Sprintf("%s-rbd-user", c.Namespace)
```

Comment:
this was specifically for use by the rbd plugin. Let's update it later with your work around the plugins